apps/docs/content/guides/functions/background-tasks.mdx

```diff
@@ -3,39 +3,59 @@ id: 'function-background-tasks'
 title: 'Background Tasks'
 description: 'How to run background tasks in an Edge Function outside of the request handler'
 subtitle: 'How to run background tasks in an Edge Function outside of the request handler'
-tocVideo: 'rSKBTJxG9VA'
```

Comment:
@thorwebdev I removed the video from the guide as the syntax has changed (no longer need to do custom events). Would you be able to do an updated version of the video?